api-1.9/src/main/java/org/openmrs/module/reporting/report/service/db/MappedDefinitionType.java

```diff
@@ -1,209 +1,208 @@
+		return Mapped.class;
+	}
+	
+	/**
+	 * @see CompositeUserType#getPropertyNames()
+	 */
+	public String[] getPropertyNames() {
+		return new String[] {"definition", "parameterMappings"};
+	}
+
+	/**
+	 * @see CompositeUserType#getPropertyTypes()
+	 */
+	public Type[] getPropertyTypes() {
+		return new Type[] { HibernateUtil.standardType("STRING"), HibernateUtil.standardType("TEXT") };
+	}
+	
+	/**
+	 * @see CompositeUserType#isMutable()
+	 */
+	public boolean isMutable() {
+		return true;
+	}
+
+	/**
+	 * @see CompositeUserType#getPropertyValue(java.lang.Object, int)
+	 */
+	public Object getPropertyValue(Object component, int property) throws HibernateException {
+		Mapped m = (Mapped) component;
+		return (property == 0 ? m.getParameterizable() : m.getParameterMappings());
+	}
+
+	/**
+	 * @see CompositeUserType#setPropertyValue(java.lang.Object, int, java.lang.Object)
+	 */
+	public void setPropertyValue(Object component, int property, Object value) throws HibernateException {
+		Mapped m = (Mapped) component;
+		if (property == 0) {
+			m.setParameterizable((Parameterizable)value);
+		}
+		else {
+			m.setParameterMappings((Map)value);
+		}
+	}
+	
+	/**
+	 * @see CompositeUserType#deepCopy(java.lang.Object)
+	 */
+	public Object deepCopy(Object value) throws HibernateException {
+		if (value == null) return null;
+		Mapped toCopy = (Mapped) value;
+		Mapped m = new Mapped();
+		m.setParameterizable(toCopy.getParameterizable());
+		m.setParameterMappings(new HashMap<String, Object>(toCopy.getParameterMappings()));
+		return m;
+	}
+
+	/**
+	 * @see CompositeUserType#nullSafeGet(ResultSet, String[], SharedSessionContractImplementor, Object)
+	 */
+	public Object nullSafeGet(ResultSet rs, String[] names, SharedSessionContractImplementor session, Object owner) throws HibernateException, SQLException {
+		String parameterizableUuid = (String) HibernateUtil.standardType("STRING").nullSafeGet(rs, names[0], session, owner);
+		if (StringUtils.isEmpty(parameterizableUuid)) { return null; }
+		String serializedMappings = (String) HibernateUtil.standardType("STRING").nullSafeGet(rs, names[1], session, owner);
+		Definition d = DefinitionContext.getDefinitionByUuid(mappedType, parameterizableUuid);
+		Map<String, Object> mappings = new HashMap<String, Object>();
+		if (StringUtils.isNotBlank(serializedMappings)) {
+			try {
+				mappings = Context.getSerializationService().deserialize(serializedMappings, Map.class, ReportingSerializer.class);
+			}
+			catch (Exception e) {
+				throw new HibernateException("Unable to deserialize parameter mappings for definition", e);
+			}
+		}
+		return new Mapped(d, mappings);
+	}
+
+	/**
+	 * @see CompositeUserType#nullSafeSet(PreparedStatement, Object, int, SharedSessionContractImplementor)
+	 */
+	public void nullSafeSet(PreparedStatement st, Object value, int index, SharedSessionContractImplementor session) throws HibernateException, SQLException {
+		String definitionUuid = null;
```

Comment:
I don't know why GitHub is showing this as a completely new file. The only thing I did was change these methods :).

Reply:
When comparing these changes locally, I am not seeing this issue.